install/kubernetes/helm/istio-remote/values.yaml

```diff
@@ -0,0 +1,7 @@
+# Use --set to specify the pilot and mixer IPs on the main Istio control plane
+global:
+  # The Istio control plane Pilot IP address
+  pilotIp: none
+
```

Comment:
ok fair point on the ip.  Seems domain is possible as long as it is resolvable.  If so, should we call it pilot endpoint? vs ip?

My main comment is whether this remote chart works when user installs istio with security enabled (istio-auth.yaml) or without security enabled (istio.yaml).  That would be good to understand and guide our users who may be consuming this remote chart.

Reply:
I don't mind calling it pilotEndpoint - that seems to make more sense.

Sure I'll test istio-auth and get back to you.  Although I have a separate PR coming shortly where that could be documented.  In any regard, I agree that case needs to work (it may not yet) and that it needs to be documented if it doesn't.

Reply:
@linsun I tested this PR against istio-auth - seems to work.  Note there is an E2E test pending which will formalize e2e testing of multicluster here: https://github.com/istio/istio/pull/4240

Reply:
@linsun note there is full documentation (which needs a little bit of rev here): https://github.com/istio/istio.github.io/pull/1139